sketch/package.json

```diff
@@ -7,34 +7,50 @@
   "jsdelivr": "dist/magentasketch.js",
   "unpkg": "dist/magentasketch.js",
   "dependencies": {
-    "@tensorflow/tfjs": "^0.13.3"
+    "@tensorflow/tfjs": "^1.0.2",
+    "@types/node": "^12.0.1",
+    "fs": "0.0.1-security",
```

Comment:
i thiiiink these are still dev dependencies, no?

Reply:
Oh! Yes, you're totally right. My mistake! Let me move those out quickly. Pardon the oopsies + thanks for catching this!

Reply:
Okie doke. This should now be sorted with : https://github.com/tensorflow/magenta-js/pull/310/commits/9bed2abad2c1a22a8e91bf22ee6eb2b48d0edfcd thank you!